### ISSUE TITLE
Variant type tables should be removed and functionality incorporated into CQL

### ISSUE THREAD

**Issue by ihodes** [hammerlab/cycledash#249]
![](http://cl.ly/image/3L2W1B3v1n2k/Screen%20Shot%202014-11-20%20at%205.18.14%20PM.png)

These don't work and should be gone.

**Comment by arahuja** [comment]
Just noticed these were gone - any reason in particular?  I guess I should be able to filter down with a query - is there a way to do that now?

**Comment by arahuja** [comment]
Looks like I can do `alternates RLIKE '^.$' and reference RLIKE '^.$'` in CQL 

Is the functionality you want to add to CQL a shorthand for this, or is the way to do it?

**Comment by ihodes** [comment]
Yeah, we definitely need to add some shorthand in, otherwise filtering is a bit onerous (sorry about that).

**Comment by ihodes** [comment]
The reason is simply that the move from front-end VCF parsing to back-end parsing dropped some features along the way.
